torch/signal/windows/windows.py

```diff
@@ -296,3 +297,83 @@ def gaussian(
                        requires_grad=requires_grad)
 
     return torch.exp(-k ** 2)
+
+
+@_add_docstr(
+    r"""
+Computes the Kaiser window
+
+The Kaiser window is defined as follows:
+
+.. math::
+    out_i = I_0 \left( \beta \sqrt{1 - \left( {\frac{i - N/2}{N/2}} \right) ^2 } \right) / I_0( \beta )
+
+Where :math:`I_0` is the zeroth order modified Bessel function of the first kind (see :func:`torch.i0`) and
```

Comment:
link to the canonical `torch.special.i0` instead of the alias (saving the reader a click)